akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -1098,7 +1108,9 @@ class DDataShardCoordinator(
   import akka.cluster.ddata.Replicator.Update
 
   private val readMajority = ReadMajority(settings.tuningParameters.waitingForStateTimeout, majorityMinCap)
-  private val writeMajority = WriteMajority(settings.tuningParameters.updatingStateTimeout, majorityMinCap)
+  private val readAll = ReadAll(settings.tuningParameters.waitingForStateTimeout)
```

Comment:
ReadAll is working well. All tests passing with it. The idea is to explore if we can use something less.